Scripts/build.swift

```diff
@@ -60,7 +60,7 @@ enum Platform: String, CustomStringConvertible {
         case .watchOS_7:
             return "OS=7.2,name=Apple Watch Series 6 - 44mm"
         case .watchOS_8:
-            return "OS=8.5,name=Apple Watch Series 6 - 44mm"
+            return "OS=8.5,name=Apple Watch Series 6 (44mm)"
```

Comment:
Whyyyyyyy

Reply:
Not sure. I've had to make this change on multiple repos. Seems something changed on these CI machines recently 😬